src/test/java/com/uid2/shared/auth/OperatorKeyTest.java

```diff
@@ -138,4 +141,72 @@ public void verifyRolesPropSetOperatorRoleAndOptoutRole() {
         Assert.assertTrue(c.getRoles().contains(Role.OPERATOR));
         Assert.assertTrue(c.hasRole(Role.OPTOUT));
     }
+
+    @Test
+    public void verifyIsPublicOperatorFlagIsOptionalForBackwardsCompatibility() {
+        final String testJson = "    {\n" +
+                "        \"key\": \"test-admin-key\",\n" +
+                "        \"name\": \"admin@uid2.com\",\n" +
+                "        \"contact\": \"admin@uid2.com\",\n" +
+                "        \"created\": 1617149276,\n" +
+                "        \"disabled\": false\n" +
+                "    }";
+
+        JsonObject jo = (JsonObject) Json.decodeValue(testJson);
+        OperatorKey c = OperatorKey.valueOf(jo);
+        Assert.assertTrue(c.getOperatorType() == OperatorType.PRIVATE);
+
+        c.setOperatorType(OperatorType.PUBLIC);
+        Assert.assertTrue(c.getOperatorType() == OperatorType.PUBLIC);
+    }
+
+    @Test
+    public void verifyIsPublicOperatorFlagOn() {
```

Comment:
Since we don't really have a boolean field IsPublicOperator, would it be better to rename the tests here?

Reply:
Thanks i have renamed these method names.